### ISSUE TITLE
isotop packery problems with percentage

### ISSUE THREAD

**Issue by edkpan** [metafizzy/isotope-packery#7]
Hello,
I'm using the packery mode with percentage and I figured out there is an issue with how divs are positionned. When my percentage isn't even (p/3) divs can't get right at all!
I suspect that positionning divs with pixels (left & top) isn't accurate enough.
Did I miss something? How may I get it right? Is there a percentage mode for positionning?

Here is the css code I use to size square and rectangular div like 1/3 or 2/3
@media only screen and (min-width: 740px) and (max-width: 959px) {
	.item{
		width: calc(100% / 3 * 1);
		padding-bottom: calc(100% / 3 * 1); }
	.item.w2 { width: calc(100% / 3 * 2); }
	.item.h2 { padding-bottom: calc(100% / 3 * 2); }
}

(Sorry for any kind of mistranslation, kiss from Paris)

**Comment by desandro** [comment]
I'm sorry to see you're having trouble with Isotope. Could you provide a **reduced test case**?  Try forking this CodePen http://codepen.io/desandro/pen/YPvZLO See [Submitting Issues in the contributing guidelines](https://github.com/metafizzy/isotope/blob/master/CONTRIBUTING.mdown#submitting-issues).

**Comment by edkpan** [comment]
Oops thank you for the guidelines - I'm new with that
It seems the problem isn't about even or odd finally. I can't get the packery going right: got gap in the layout...
http://codepen.io/anon/pen/MYXpBb

**Comment by edkpan** [comment]
Ok, the heart of the problem is the box-sizing propriety.
It doesn't place border inside divs as supposed to: don't know why. Any idea?
So this issue was simply " solved " by replacing the border propriety by an outline.
http://codepen.io/anon/pen/qEKjgq
Anyways, thank you very much for this awesome plugin!

**Comment by desandro** [comment]
Thanks for that! This looks to be related to how the borders throw off the size calculation. If borders are removed, I don't see any gaps http://codepen.io/desandro/pen/vErxbb

I investigated this and it appears you came into a rare browser bug, when you use `padding-bottom`, `calc`, and `box-sizing: border-box`. See http://codepen.io/desandro/pen/YPvZgd

To resolve this, I adjusted add `padding-bottom` values to account for the 2px border size. See http://codepen.io/desandro/pen/OPEjeL
